src/analyze/attw.ts

```diff
@@ -5,12 +5,14 @@ import {
 } from '@arethetypeswrong/core';
 import {groupProblemsByKind} from '@arethetypeswrong/core/utils';
 import {filterProblems, problemKindInfo} from '@arethetypeswrong/core/problems';
-import {ReportPluginResult} from '../types.js';
+import {ReportPluginResult, type Options} from '../types.js';
 import type {FileSystem} from '../file-system.js';
 import {TarballFileSystem} from '../tarball-file-system.js';
 
 export async function runAttw(
-  fileSystem: FileSystem
+  fileSystem: FileSystem,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  _?: Options
```

Comment:
`no-unused-vars` should be setup to ignore `_` prefixed names, and this should be called `_options`